### PR TITLE
Update sans.rst

### PR DESCRIPTION
**Pull request by AnthonyLim23**
Just added a few fullstops and changed a few words

Description of work.

**To test:**

Fixes #xxxx.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?

- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by AntonPiccardoSelg** [comment]
@AnthonyLim23 
`Fix for beam center finder. ` seems better than`Fixed for beam center finder. `. What do you think?

**Comment by AnthonyLim23** [comment]
yes you are right, I will change that. I just put fixed to be consistent

**Comment by AntonPiccardoSelg** [comment]
@AnthonyLim23  When you make a change and push it, it is recommended to cancel the builds on previous commits. This can free up considerable resources. Thanks

**Comment by AnthonyLim23** [comment]
I am not sure how to do that from github. I do not have access to anything else from home.

**Comment by AntonPiccardoSelg** [comment]
notes look fine. Once tests pass, please :shipit:

File: docs/source/release/v3.8.0/sans.rst [hunk 1]
```diff
@@ -9,25 +9,25 @@ Features
...
-- Fix for beam center finder
-- Fix loading of multiperiod event files
-- Fix period selection when loading multiperiod files.
-- Fix process note for SaveCanSAS1D
-- Fix output names for batch processing
-- Fix the loading of RKH files
-- Fix wrong initial position of LARMOR data in the beam centre finder
-- Allow loading of CanSAS data without error data
-- Fix saving CanSAS with transmission data from the ISIS SANS GUI
+- Fix for beam center finder.
```
Comment (AndreiSavici) [inline docs/source/release/v3.8.0/sans.rst]: `Fixed` instead of `Fix`? Just to be consistent

Comment (AndreiSavici) [inline docs/source/release/v3.8.0/sans.rst]: Sorry, I did not carefully read the previous comments

**Pull request merged**